lib/internationalization/ISO_handler.fz

```diff
@@ -0,0 +1,27 @@
+# This file is part of the Fuzion language implementation.
+#
+# The Fuzion language implementation is free software: you can redistribute it
+# and/or modify it under the terms of the GNU General Public License as published
+# by the Free Software Foundation, version 3 of the License.
+#
+# The Fuzion language implementation is distributed in the hope that it will be
+# useful, but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public
+# License for more details.
+#
+# You should have received a copy of the GNU General Public License along with The
+# Fuzion language implementation.  If not, see <https://www.gnu.org/licenses/>.
+
+
+# -----------------------------------------------------------------------
+#
+#  Tokiwa Software GmbH, Germany
+#
+#  Source code of Fuzion standard library feature internationalization.ISO_handler
+#
+# -----------------------------------------------------------------------
+
+# ISO_handler -- provides data for ISO 8601 format
+#
+public ISO_handler : int_handler is
+    redef date(d i32,m i32,y i32) => "{y}-{m}-{d}"
```

Comment:
please keep one empty line before and after feature declarations and add a comment for each feature, e.g., 

```suggestion
# ISO_handler -- provides data for ISO 8601 format
#
public ISO_handler : int_handler is

    # format date using ISO formatting rules, see https://en.wikipedia.org/wiki/ISO_8601
    # 
    # ex. `date 2024 4 12` results in `2024-04-12`.
    #
    redef date(d i32,m i32,y i32) 
    => 
      "{y}-{m}-{d}"
```